src/layout.js

```diff
@@ -527,7 +529,12 @@ export function applyStaticLayout(element) {
     const intrinsicSizer = sizer.firstElementChild;
     intrinsicSizer.setAttribute(
       'src',
-      `data:image/svg+xml;charset=utf-8,<svg height="${height}" width="${width}" xmlns="http://www.w3.org/2000/svg" version="1.1"/>`
+      fixIeIntrinsic
```

Comment:
Since this is specific to IE only, could we wrap the logic here with an `IS_ESM` check and not pass around the argument?